scripts/provision-kind-cluster.sh

```diff
@@ -24,18 +23,17 @@ K8_1_16="kindest/node:v1.16.9@sha256:7175872357bc85847ec4b1aba46ed1d12fa054c83ac
 K8_1_15="kindest/node:v1.15.11@sha256:6cc31f3533deb138792db2c7d1ffc36f7456a06f1db5556ad3b6927641016f50"
 K8_1_14="kindest/node:v1.14.10@sha256:6cd43ff41ae9f02bb46c8f455d5323819aec858b99534a290517ebc181b443c6"
 
-K8_VERSION="$K8_1_16"
-
 USAGE="
 Usage:
-  $(basename "$0") CLUSTER_NAME [-v K8S_VERSION]
+  $(basename "$0") CLUSTER_NAME
 
 Provisions a KinD cluster for local development and testing.
 
-Example: $(basename "$0") my-test -v 1.16
+Example: export K8S_VERSION=1.18; $(basename "$0") my-test
```

Comment:
I'd remove the `export K8S_version=1.18` here, since it's an optional argument and described in the `Environment variables:` section in the USAGE already.

Reply:
done :)